test-cases/longevity/longevity-50GB-4days-authorization-and-tls-ssl.yaml

```diff
@@ -20,7 +20,7 @@ n_monitor_nodes: 1
 
 instance_type_db: 'i3.4xlarge'
 # Seems the c5.xlarge type is small for this load - I receive OOM on 2 loaders
-instance_type_loader: 'c5.2xlarge'
```

Comment:
Why do you change this? can you add reason to commitlog?

Reply:
@ShlomiBalalis explain why in the commit message.

The reason is that we get spot termination for c5 very easily.